packages/shared/pkg/utils/map.go

```diff
@@ -0,0 +1,9 @@
+package utils
+
+func Map[T any, U any](input []T, f func(T) U) []U {
```

Comment:
also here, I think `Map` is kinda overloaded word. Just we don't have to scan the code